hack/release/make-release-tag.sh

```diff
@@ -49,6 +48,13 @@ for example in examples/contour/03-envoy.yaml examples/contour/03-contour.yaml ;
         "$example"
 done
 
+# Update the certgen job name to the new version.
+for example in examples/contour/02-job-certgen.yaml ; do
+    run::sed \
+        "-es|contour-certgen-master|contour-certgen-$NEWVERS|" \
```

Comment:
I'm thinking we should also look for `$OLDVER`, like https://github.com/projectcontour/contour/pull/2701/files#diff-69f6aa2773b15b9355c06df3155fd0afR47, for consistency - probably relevant for patch releases being created off an existing release branch?

Reply:
I guess I don't understand why we have $OLDVER in the script. If you made a patch from say v1.5.0 --> v1.5.1, you'd still want $NEWVER (e.g. v1.5.1) as the tag right?

Reply:
right, but by then the release-1.5 branch would already have `v1.5.0` in its YAML, so you'd need to sed on that, rather than `master` - that's my understanding anyway.

Reply:
Ahh ok that makes sense. Let me check on my script, it won't work like that.

Reply:
I think you just want:

```
run::sed \
        "-es|contour-certgen-master|contour-certgen-$NEWVERS|" \
        "-es|contour-certgen-$OLDVERS|contour-certgen-$NEWVERS|" \
        "$example"
```

Reply:
ok updated @skriss